.github/workflows/release.yaml

```
@@ -0,0 +1,69 @@
+name: Release
```

I find it useful to see the latest released versions in the list of workflow runs.
We could use something like:
```
run-name: release ${{ inputs.tag }}
```

---

Sounds good! let's add it
```suggestion
---
name: Release

run-name: release ${{ inputs.tag }}
```